portfolio/src/main/webapp/script.js

```diff
@@ -13,20 +13,28 @@
 // limitations under the License.
 
 /**
- * Show/hide the description inside the element which has title as id.
- */
-function toggleContent(sectionId) {
-  if (document.getElementById(sectionId) == null) {
-      return;
+* Insert/remove the class in the element.
+*/
+function toggleClass(element, classAttribute) {
+  if (!element || typeof element.className !== "string" || !classAttribute || typeof classAttribute !== "string") {
+    return;
+  }
```

Comment:
Good call to validate your input.
To make it more readable, I would avoid having so many conditions in a single if statement however.
Consider checking each input value separately.